modules/xmpp/JingleSessionPC.js

```diff
@@ -575,19 +573,17 @@ export default class JingleSessionPC extends JingleSession {
         // The signaling layer will bind it's listeners at this point
         this.signalingLayer.setChatRoom(this.room);
 
-        if (!this.isP2P && options.enableLayerSuspension) {
+        if (!this.isP2P) {
             // If this is the bridge session, we'll listen for
-            // IS_SELECTED_CHANGED events and notify the peer connection
-            this.rtc.addListener(RTCEvents.IS_SELECTED_CHANGED,
-                isSelected => {
-                    this.peerconnection.setIsSelected(isSelected);
-                    logger.info('Doing local O/A due to '
-                        + 'IS_SELECTED_CHANGED event');
-                    this.modificationQueue.push(finishedCallback => {
-                        this._renegotiate()
-                            .then(finishedCallback)
-                            .catch(finishedCallback);
-                    });
+            // SENDER_VIDEO_CONSTRAINTS_CHANGED events and notify the peer connection
+            this.rtc.addListener(RTCEvents.SENDER_VIDEO_CONSTRAINTS_CHANGED,
+                idealHeight => {
+                    logger.info(`${this} received remote ideal frame height: ${idealHeight}`);
+                    this.remoteRecvMaxFrameHeight = idealHeight;
+                    this.eventEmitter.emit(
+                        MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED,
+                        this,
+                        idealHeight);
```

Comment:
Looks like the height passed with this event is not used in the function invoked on the listener. JVB case is fine since you are setting `this.remoteRecvMaxFrameHeight` to this new height value received but @paweldomas I am curious as to how this works in the p2p case. Maybe I am missing something.

Reply:
that's a great point Jaya.. thanks for pointing that out. I'll try to clean this up tomorrow.

Reply:
It looks like the resolution argument was added to facilitate the tests. f6a83bb4 should alleviate this but I don't know how to run the tests. @paweldomas could you please confirm that f6a83bb4 is an acceptable solution?

Reply:
yes, it's good